### ISSUE TITLE
remember column width

### ISSUE THREAD

**Issue by danielkrajnik** [cboxdoerfer/fsearch#325]
Everything remembers column width after closing the program. It would be nice if one prefers for example really wide path column.

**Comment by kajodowe** [comment]
Yes, this would be my number one feature request!

**Comment by cboxdoerfer** [comment]
Do you have `Preferences -> Rememeber column configuration` enabled?

**Comment by danielkrajnik** [comment]
@cboxdoerfer yes, "rememeber column configuration" was enabled already, but it doesn't remember their width. It remembers, which and in what order columns were set, but no the width itself.

**Comment by begtognen** [comment]
@cboxdoerfer That's true for me as well - "remember column configuration" is enabled, but doesn't remember width.

**Comment by lepa22** [comment]
@cboxdoerfer The same for me. Also, I cannot change column width, unless some search results are already displayed. I don't know if there's a relation between these two.

**Comment by kajodowe** [comment]
It seems to work now with the latest update -- the width is remembered along with the column configuration. Thanks very much Christian!

**Comment by danielkrajnik** [comment]
Great news, happy new year!

**Comment by begtognen** [comment]
That's great! My apologies if this is the wrong place to ask, but I'm not sure how to get the latest update - I did follow the instructions in the link above but I think I might still have the wrong version as it doesn't remember column width. In the About dialog box it says "0.1beta4".

**Comment by kajodowe** [comment]
begtognen -- see the post just above yours from danielkrajnik -- you need to have the PPA he identifies loaded into your Package Manager -- the one ending in '/fsearch-daily' -- hopefully Christian will respond here and help you if have additional questions.

I copied these instructions below from Christian's page at https://launchpad.net/~christian-boxdoerfer/+archive/ubuntu/fsearch-daily. This should fix it for you.

--------------------------------------------------------
> Adding this PPA to your system
You can update your system with unsupported packages from this untrusted PPA by adding ppa:christian-boxdoerfer/fsearch-daily to your system's Software Sources. (Read about installing)

sudo add-apt-repository ppa:christian-boxdoerfer/fsearch-daily
sudo apt-get update
----------------------------------------------------------

**Comment by danielkrajnik** [comment]
followed by **sudo apt install fsearch**

**Comment by begtognen** [comment]
Thanks very much - it works perfectly now. So happy to have the column width remembered!
